src/librustdoc/html/static/js/settings.js

```diff
@@ -91,10 +94,172 @@
                 changeSetting(ev.target.name, ev.target.value);
             });
         });
-        document.getElementById("back").addEventListener("click", function() {
-            history.back();
+    }
+
+    function buildSettings(settings) {
```

Comment:
`buildSettings`, `buildSettingsPage`, and `createSettingsPage` should each have comments explaining what they do and what they return. Also, all three of these have very similar names ("build" and "create" are nearly synonyms). Can you explain the distinctions between them?